common/src/main/scala/org/apache/celeborn/common/network/CelebornRackResolver.scala

```diff
@@ -32,6 +32,8 @@ import org.apache.celeborn.common.util.CelebornHadoopUtils
 
 class CelebornRackResolver(celebornConf: CelebornConf) extends Logging {
```

Comment:
How about moving this class to master module? I found that our common module contains too much content.

Reply:
> How about moving this class to master module? I found that our common module contains too much content.

Also need use this in LifecycleManager's reallocate logic.

Reply:
Why LifecycleManager will use this? It will be too heavy for client. Maybe we can call offerSlots to revive if rackawareness is enabled.

Reply:
> Why LifecycleManager will use this? It will be too heavy for client. Maybe we can call offerSlots to revive if rackawareness is enabled.

Current code add networkLocation into worker info, LifecycleManager direct use the information